src/ontology/edits/oeo-model.omn

```diff
@@ -4399,9 +4400,9 @@ Class: OEO_00410032
 
     Annotations: 
         OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1962
-pull request: https://github.com/OpenEnergyPlatform/ontology/pull/2006",
-        <http://purl.obolibrary.org/obo/IAO_0000115> "A high soil quality role is a soil quality role inherent in a region of relevance reflecting that the soil, that is part of the bearer, has a high soil quality."@en,
-        rdfs:label "high soil quality role"@en
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/2041",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An arable land with high soil quality role is an arable land role inherent in an arable land reflecting that the soil, that is part of the bearer, has a high soil quality."@en,
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "An arable land with high soil quality role is a role of an arable land (region of relevance) reflecting that the soil, that is part of the bearer, has a high soil quality."@en,
        rdfs:comment "An arable land has some soil as part, whose quality can be categorized respecting all current positive and negative properties with regard to soil utilization and soil functions. See also the soil quality definition from GEMET https://www.eionet.europa.eu/gemet/en/concept/7884",
```
For better readability, I suggest to deviate slightly from the Arestotelian definition format.